plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ContractDescriptionConversionVisitor.kt

```diff
@@ -196,13 +207,14 @@ class ContractDescriptionConversionVisitor(
         embeddedVarByIndex(parameterIndex)
 
     private fun KtValueParameterReference<ConeKotlinType, ConeDiagnostic>.getTargetParameter(data: ContractVisitorContext): FirBasedSymbol<*> {
-        return resolveByIndex(
+        return resolveByIndex<FirBasedSymbol<*>>(
```

Comment:
Is this necessary?

Reply:
You're right, it is not necessary anymore!